netutils/netinit/netinit.c

```diff
@@ -1,9 +1,10 @@
 /****************************************************************************
  * apps/netutils/netinit/netinit.c
  *
- *   Copyright (C) 2010-2012, 2014-2016, 2019 Gregory Nutt. All rights
+ *   Copyright (C) 2010-2012, 2014-2016, 2019-2020 Gregory Nutt. All rights
  *     reserved.
- *   Author: Gregory Nutt <gnutt@nuttx.org>
+ *   AuthorS: Gregory Nutt <gnutt@nuttx.org>
+ *            David Sidrane <david.sidrane@nscdg.com>
  *
  * Redistribution and use in source and binary forms, with or without
  * modification, are permitted provided that the following conditions
```

Comment:
Why not an Apache 2.0 header?

Reply:
I will take that as you grant, I will update it.

Reply:
Done in 22493084fcb81fdd055eb1075f6636a0475721dd